src/main/java/racingcar/OutputView.java

```diff
@@ -1,17 +1,29 @@
 package racingcar;
 
+import java.util.List;
+
 public class OutputView {
-    public void printCarPosition(int carPosition) {
-        for (int i = 0; i <= carPosition; i++) {
+    public void printCarPosition(Car car) {
+        // 입력변수를 Car 하나로 두면 깔끔하긴 한데 캡슐화가 깨지는 거 같음
+        System.out.print(car.name() + ": ");
+        for (int i = 0; i <= car.position(); i++) {
             System.out.print('-');
         }
         System.out.print('\n');
     }
 
     public void printCarGroupPosition(CarGroup carGroup) {
-        for (int carIndex = 0; carIndex < carGroup.carGroupLength(); carIndex++) {
-            printCarPosition(carGroup.position(carIndex));
+        for (int carIndex = 0; carIndex < carGroup.size(); carIndex++) {
+            printCarPosition(carGroup.car(carIndex));
         }
         System.out.println("==========");
     }
+
+    public void printWinner(CarGroup carGroup) {
+        List<Car> winners = Winners.findWinners(carGroup.carGroup());
```

Comment:
View에서 Winners에게 메시지를 보내지 말고 우승자 목록을 인자로 받아 출력만 하면 어떨까요?
우승자를 구하는 로직을 호출하는 코드는 Racing 객체에서 하면 좋을 것 같아요
```suggestion
    public void printWinner(List<String> winnerNames) {
```

Reply:
말씀하신 것처럼 View는 winners에 의존하면 안된다는 생각이 들어 수정하였습니다!